### DISCUSSION TITLE
tgcf-web doesn't want to run

### DISCUSSION THREAD

**Discussion by irovder** [aahnik/tgcf#367] [Q&A] [answered]
I installed tgcf in Raspberry Pi OS, Python3.9 venv.
`tgcf --version` gives `0.2.11.post0`
but `tgcf-web` gives `-bash: tgcf-web: command not found`
What could be wrong?

**Comment by tissole** [comment, marked as answer]
You have the old version of tgcf, there is no web-ui in that. Most likely you got the old one because the new script has python 3.10 as dependency, so the pip installed the old version which works on python>3.8. You have to install python 3.10 if you want the new script. Don't update to python 3.11, it won't work.

**Reply by irovder** [reply]
Thanks I'll try it.
